src/main/java/org/springframework/cloud/deployer/spi/kubernetes/AbstractKubernetesDeployer.java

```diff
@@ -258,7 +258,10 @@ PodSpec createPodSpec(AppDeploymentRequest appDeploymentRequest) {
 		if (initContainer != null) {
 			podSpec.addToInitContainers(initContainer);
 		}
-
+		List<Container> additionalContainers = this.deploymentPropertiesResolver.getAdditionalContainers(deploymentProperties);
+		for (Container additionalContainer: additionalContainers) {
+			podSpec.addToContainers(additionalContainer);
```

Comment:
looks like theres a `addAllToContainers` that could be used rather than the loop here